### ISSUE TITLE
/rd (ew, entitywipe) clearing counter

### ISSUE THREAD

**Issue by mayokek** [TotalFreedom/TotalFreedomMod#1886]
/rd (ew, entitywipe) does work fine but the only issue with it is that the counter kept saying 0 entities removed. No matter what been dropped, it would wipe it but doesn't count up.

**Comment by LeohPlayzMC1** [comment]
Yeah you're right, this is a bug that's actually been around for a short while because I've noticed Admins on TotalFreedom complaining about it. 

I'll patch it and submit a non-bugged version of it to Prozza and/or Wild.

**Comment by ghost** [comment]
Yes, just tested this. I was about to report it. 

![2016-11-28_08 53 02](https://cloud.githubusercontent.com/assets/20630375/20670865/02861eae-b549-11e6-8e6a-cba83e1692d5.png)

**Comment by Wild1145** [comment]
Could someone please confirm this is still in fact an issue? I cant see any reason it would be breaking...

**Comment by hitzzaf** [comment]
bump - sstill not fixed

**Comment by Wild1145** [comment]
@hitzzaf Please do not "Bump" issues, it simply wastes my time in trying to deal with the issues on this Github. The issue is still open and when the bug is resolved, this issue will be closed as a result. Until then please only post to add additional information / similar, bumping is not required.
